index.js

```diff
@@ -574,26 +578,32 @@ let runHook = function (hookFn, origFn, before, after, repeatTest = 0) {
  * @return {Function}             wrapped test function
  */
 let runSpec = function (specTitle, specFn, origFn, repeatTest = 0) {
+    // filter out arguments passed to specFn, don't allow callbacks
+    // as there is no need for user to call e.g. `done()`
+    const filterSpecArgs = (args) => {
```

Comment:
Let's define this function somewhere at the top of the file and reuse it everywhere in `runHook` and here